pom.xml

```diff
@@ -1426,6 +1426,7 @@
         <scope>test</scope>
       </dependency>
       <!-- Managed up to match Hadoop in HADOOP-16530 -->
+      <!-- When upgrading `xercesImpl` version, also need to change the version definition in `SparkBuild#DependencyOverrides`. -->
```

Comment:
Thank you, @LuciferYang . Could you split this into two lines?

Reply:
done